### DISCUSSION TITLE
User Submissions for Video

### DISCUSSION THREAD

**Discussion by mega12345mega** [mega12345mega/NBT-Editor#47] [Announcements]
I'm planning to make a video for the mod which shows off some of the features! I'm going to post it on the Curseforge and Modrinth pages, so potential users can see different things they can do and hopefully be inspired.

I thought that it might be cool to include epic items from the community. If you would like to have your item(s) featured in the video, please comment them here. Use `/nbt export get` (small items) or `/nbt export file` (items with lots of data) and send the result, which I can then import into the game. If you would like to be credited by your Minecraft username instead of your Github username, also include that in your message.

Along with the video, I'm going to distribute a `.zip` of all of the featured items. If you don't want me to distribute your item, just mention that. (You may also want to use Curseforge's private message feature in that case.)

Thanks!

**Comment by serainox420** [comment]
[client_chest.zip](https://github.com/mega12345mega/NBT-Editor/files/11470207/client_chest.zip)
Hey, I made some funny and crazy items using NBT Editor back in the days, tho I don't have time to go trough them and sort out only the best examples, thus Im including my entire client_chest with items naturally saved in page.nbt 
(sorry for not being compliant with your query but there's my time issue again)

Im sure you will be able to find dozen cool items to use as example :smile: 

Don't mind it being bit of a mess hah

**Comment by ghost** [comment]
[2023-05-14_10_21_28.zip](https://github.com/mega12345mega/NBT-Editor/files/11471359/Sus.Stew_2023-05-14_10_21_28.zip)

lol here's something

**Comment by KadTheHunter** [comment]
[KadsCollection.zip](https://github.com/mega12345mega/NBT-Editor/files/11503728/KadsCollection.zip)

This is a full inventory (including hotbar), so make sure you have room :D

The only things actually made by _me_ is `Kaddicus Doll` and `Kaddicus was here`, the rest is just things I've collected and shoved in my clientchest.

MC IGN is Kadalyst, unsure if you should credit me for all or just mine tho.

One very important note: the sign called NBTELag will actually lag NBT Editor when opened in it. Still don't understand all the details of _why_, but it sure is funny.

If you want more items, or can include kits, check out [The Shulker Archives](https://kadthehunter.github.io/ShulkerArchives/index.html), I've been working to archive Shulker Kits, Items and Books for months now, and have a fairly massive collection of kits (havent gotten to items/books too much yet, but there are some).

**Comment by mega12345mega** [comment]
I've finally created the video! Here's the link if you're interested: [https://www.youtube.com/watch?v=zNA5rkXP8Sc](https://www.youtube.com/watch?v=zNA5rkXP8Sc)
Thanks to everyone who submitted items!

@KadTheHunter I thought the Shulker Archives website was cool, so I mentioned it at the end of the video!
